src/devices/tfa_30.3211.c

```diff
@@ -0,0 +1,101 @@
+#include "rtl_433.h"
+#include "util.h"
+#include "data.h"
+
+/*
+ TFA 30.3211.02 
+ 
+ 1970us pulse with variable gap (third pulse 3920 us)
+ 
```

Comment:
we need to investigate if that pulse is preamble/sync or some bitstuffing

Reply:
sample g008.... contains long third pulse (above 79 %)

I tested my code from 30% to 98% and -9.7° to 21.7°